src/ServiceControl/Operations/TransportMessageCleaner.cs

```diff
@@ -1,14 +0,0 @@
-﻿using System;
-
-namespace ServiceControl.Operations
-{
-    using NServiceBus;
-
-    static class TransportMessageCleaner
-    {
-        public static void CleanForForwarding(TransportMessage message)
-        {
-            message.TimeToBeReceived = TimeSpan.MaxValue;
```

Comment:
How is TTBR handled in V7? Is it only applied if there's a delivery constraint or is it implemented as a header? This used to cause a problem with sending messages to the Audit Log Queue that got removed due to TTBR.

Reply:
This code was never called. Maybe it was not due to the refactoring I'll double check